### PR TITLE
Remove get managed bean

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/test/java/de/sub/goobi/forms/RulesetFormIT.java [hunk 1]
```diff
@@ -1,50 +0,0 @@
-/*
```
[the remark this answers is not in the document]
Comment (Beacze) [reply Kitodo/src/test/java/de/sub/goobi/forms/RulesetFormIT.java]: Those tests are in this moment covered by selenium tests - it is more reliable to check if redirect was correct in selenium than checking string value here.

Other reason was that now RulesetForm uses constructor with inject and instantiating it was not that simple again.

File: Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java [hunk 2]
```diff
@@ -108,18 +108,17 @@ public void daysClick() {
      *         in faces_config.xml
      */
     public String createProcessesClick() {
-        ProzesskopieForm prozesskopieForm = (ProzesskopieForm) Helper.getManagedBeanValue("ProzesskopieForm");
-        if (!prozesskopieForm.isContentValid(false)) {
+        //TODO: find more elegant way to validate content from processCopyForm - avoid injection
+        /*if (!processCopyForm.isContentValid(false)) {
```
[the remark this answers is not in the document]
Comment (Beacze) [reply Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java]: Method is not used in any case. GranularityForm in that moment is whole not used.

[the remark this answers is not in the document]
Comment (henning-gerhardt) [reply Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java]: Ok.

File: Kitodo/src/test/java/de/sub/goobi/forms/RulesetFormIT.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (henning-gerhardt) [reply Kitodo/src/test/java/de/sub/goobi/forms/RulesetFormIT.java]: Ok even it is bad to remove tests ;)

**Pull request merged**